specs/deneb/p2p-interface.md

```diff
@@ -149,6 +149,7 @@ This topic is used to propagate signed blob sidecars, where each blob index maps
 The following validations MUST pass before forwarding the `signed_blob_sidecar` on the network, assuming the alias `sidecar = signed_blob_sidecar.message`:
 
 - _[REJECT]_ The sidecar is for the correct subnet -- i.e. `compute_subnet_for_blob_sidecar(sidecar.index) == subnet_id`.
+- _[REJECT]_ The sidecar's index is consistent with `MAX_BLOBS_PER_BLOCK` -- i.e. `sidecar.index < MAX_BLOBS_PER_BLOCK`.
```

Comment:
actualy the previous check itself assures that this is not unbounded : `compute_subnet_for_blob_sidecar(sidecar.index) == subnet_id` so this is a redundant check

Reply:
Not true though because `compute_subnet_for_blob_sidecar` calculates the subnet_id using `mod`, so for example for subnet 0 and index 100, `compute_subnet_for_blob_sidecar(100) == 0` thus valid according to the rule.

Reply:
ahh in that case do we want it to be hardfork independent `MAX_BLOB_COMMITMENTS_PER_BLOCK` or `MAX_BLOBS_PER_BLOCK`?

Reply:
I believe @tbenr and @StefanBratanov are correct and that this should be bounded

There is an equivalent condition for beacon_attestation subnets  here -- https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/p2p-interface.md#beacon_attestation_subnet_id

> REJECT] The committee index is within the expected range -- i.e. data.index < get_committee_count_per_slot(state, data.target.epoch)

I would argue that this should be the first condition -- before the second condition that calls `compute_subnet_for_blob_side_car`. Tossing out junk first (and mirroring the order in the attestation subnet)

Reply:
> I would argue that this should be the first condition ...

True, we could save the function call but currently computes just a modulo.
Teku will perform this in this order in any case because we do the `subnet_id` check closer to the networking layer and the actual gossip validation run on top of it.